README.md

```diff
@@ -1,32 +1,35 @@
 # Logsearch
 
-A scalable stack of [Elasticsearch](http://www.elasticsearch.org/overview/elasticsearch/),
-[Logstash](http://www.elasticsearch.org/overview/logstash/), and
-[Kibana](http://www.elasticsearch.org/overview/kibana/) for your
-own [BOSH](http://docs.cloudfoundry.org/bosh/)-managed infrastructure.
+A scalable stack of [Elasticsearch](https://www.elastic.co/elasticsearch), [Logstash](https://www.elastic.co/logstash), and [Kibana](https://www.elastic.co/kibana) for your own [BOSH](https://bosh.io/docs)-managed infrastructure.
+
+![logsearch-scheme](docs/img/logsearch.png)
 
 ## BREAKING CHANGES
 
-Logsearch < v23.0.0 was based on Elasticsearch 1.x and Kibana 3.
+### Logsearch v211 is based on Elastic stack version 7
+In v211.1.0 basic cluster security features were implemented using [Security](https://opendistro.github.io/for-elasticsearch-docs/docs/install/plugins/) plugin from OpenDistro Elasticsearch implementation. For better handling of these features, a following changes was made:
 
-Logsearch > v200 is based on Elasticsearch 2.x and Kibana 4.
+- Additional Elasticsearch job has been colocated on **Maintenance** instance. This allows secure communication over localhost for all singletons also colocated there (all singletons have been unlinked from any remote Elasticsearch cluster, and bound to local one).
```

Comment:
we need to remember to update our deployments to match (if we don't do this already...)

Reply:
The job is already there: https://github.com/cloud-gov/cg-deploy-logsearch/blob/e6f520c3c17d96b9653179ef81b9919e7cf0ceae/logsearch-jobs.yml#L69